### ISSUE TITLE
API extensions is now out of beta

### ISSUE THREAD

**Issue by jaxxstorm** [pulumi/pulumi-eks#650]
EKS has updated their k8s version, so now we get:

```
Warning: apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
```

https://github.com/pulumi/pulumi-eks/blob/0ac1e0ce9faa5339c620d4bc65d42f7dfa0ee238/nodejs/eks/cni/aws-k8s-cni.yaml

**Comment by viveklak** [comment]
Yup - we should definitely switch away from any deprecated versions in our components.

**Comment by stack72** [comment]
Closed via #693
